contributors/design-proposals/resource-management/device-plugin.md

```diff
@@ -131,7 +123,7 @@ through the gRPC apis.
 When setting up the cluster the admin knows what kind of devices are present
 on the different machines and therefore can select what devices to enable.
 
-The cluster admin knows his cluster has NVIDIA GPUs therefore he deploys
+E.g., the cluster admin knows his cluster has NVIDIA GPUs therefore he deploys
```

Comment:
Nit: docs style recommends avoiding latin-isms. Could be expanded in English as "For example, ...".

Reply:
Another nit while editing this section; avoid use of gendered pronouns in docs. This sentence could be rewritten without them as "For example, the cluster admin knows the cluster has NVIDIA GPUs and deploys the NVIDIA device plugin ..."